src/sora-card-kyc-tutorial.md

```diff
@@ -0,0 +1,124 @@
+---
+title: How to Sign Up for SORA Card - A Comprehensive Guide | SORA Docs
+head:
+  - - meta
+    - name: description
+      content: Discover a step-by-step walkthrough guide on how to sign up for a SORA Card, undergo the KYC process, and ensure ease of mind. Learn about the channels, documentation, and seamless registration process.
+
+  - - meta
+    - name: keywords
+      content: SORA Card sign up, KYC process, SORA Card application channels, required documents for SORA Card, registration and KYC flow, SORA Wallet mobile app, XOR purchase, seamless sign up process
+---
+
+# **How to Sign Up for SORA Card, a Walkthrough**
+
+**In this article, you will find a walkthrough to sign up for SORA Card and undergo the KYC process.**
```

Comment:
```suggestion
```
there is a similar "in this article you will find" line below so I think you can remove this